frontend/src/views/camp/CampProgram.vue

```diff
@@ -127,6 +252,37 @@ export default {
         : this.$tc('views.camp.campProgram.reminderLockedCreate')
       this.showReminder = true
     },
+    match(scheduleEntry) {
+      return (
+        this.filteredPropertiesCount === 0 ||
+        ((this.filter.category === null ||
+          this.filter.category.length === 0 ||
+          this.filter.category.includes(
+            scheduleEntry.activity().category()._meta.self
+          )) &&
+          (this.filter.responsible === null ||
+            this.filter.responsible.length === 0 ||
+            this.filter.responsible?.every((responsible) =>
+              scheduleEntry
+                .activity()
+                .activityResponsibles()
+                .items.map((responsible) => responsible.campCollaboration()._meta.self)
+                .includes(responsible)
+            ) ||
+            (this.filter.responsible[0] === 'none' &&
+              scheduleEntry.activity().activityResponsibles().items.length === 0)) &&
+          (this.filter.progressLabel === null ||
+            this.filter.progressLabel.length === 0 ||
+            this.filter.progressLabel?.includes(
+              scheduleEntry.activity().progressLabel?.()._meta.self ?? 'none'
+            )))
+      )
+    },
```

Comment:
Could this logic be shared with the dashboard?

Reply:
Yes. But I didn't because they are slightly different and I did not want to add additional complexity and they might change in the future again.